### PR TITLE
Run Julia Pkg build

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: deps/build.jl [hunk 1]
```diff
@@ -17,8 +17,20 @@ makeopts = ["--", "-j", "$(Sys.CPU_CORES+2)"]
...
+    maker = "make"
+end
+
+# Build on windows
+
+@static if is_windows()
+  if haskey(ENV, "BUILD_ON_WINDOWS") && ENV["BUILD_ON_WINDOWS"] == "1"
+    saved_defaults = deepcopy(BinDeps.defaults)
+    empty!(BinDeps.defaults)
+    append!(BinDeps.defaults, [BuildProcess])
```
Comment (tkelman) [inline deps/build.jl:29]: you should pop this back when done

Comment (SylvainCorlay) [reply deps/build.jl:29]: done!

**Comment by SylvainCorlay** [comment]
I am still confused why `@BinDeps.install Dict([(:tensors, _l_tensors)])` would not work on windows while
 - it does on linux
 - `tensors.dll` is properly generated and installed in `/usr/lib`.

**Comment by tkelman** [comment]
dlls belong in usr/bin on windows

**Comment by SylvainCorlay** [comment]
Weird that `nmake install` would place them in `lib`! Thanks.

**Comment by tkelman** [comment]
that's probably a bug in your cmake config, ref https://cmake.org/cmake/help/v3.8/command/install.html#installing-targets

**Comment by SylvainCorlay** [comment]
OK, thanks for your help. Now trying to use the cmake variables `CMAKE_INSTALL_BINDIR` in https://github.com/QuantStack/xtensor-julia/pull/31/files

**Comment by SylvainCorlay** [comment]
So now, the dll goes to bin as it should!

Although I still get the error

```
[100%] Built target tensors
Install the project...
-- Install configuration: "Release"
-- Installing: C:/Users/appveyor/.julia/v0.5/Xtensor/deps/usr/lib/tensors.lib
-- Installing: C:/Users/appveyor/.julia/v0.5/Xtensor/deps/usr/bin/tensors.dll
===============================[ ERROR: Xtensor ]===============================
LoadError: Provider BinDeps.BuildProcess failed to satisfy dependency tensors
while loading C:\Users\appveyor\.julia\v0.5\Xtensor\deps\build.jl, in expression starting on line 85
================================================================================
================================[ BUILD ERRORS ]================================
```

**Comment by tkelman** [comment]
probably need to give the non-lib-prefixed name as a possible alias

**Comment by SylvainCorlay** [comment]
Hum that was not it...

**Comment by tkelman** [comment]
Is there a dlopen failure then?

**Comment by SylvainCorlay** [comment]
I honestly don't know. It is the same error message with no further details.
I will be AFK for a few hours and will get back to it after.

**Comment by tkelman** [comment]
see https://www.appveyor.com/docs/how-to/rdp-to-build-worker if you don't have a local windows system to test on

**Comment by SylvainCorlay** [comment]
Thanks. Yeah I don't have a proper windows development machine.

**Comment by SylvainCorlay** [comment]
It is fixed! Cf #31

**Pull request merged**